server/clip_server/model/clip_model.py

```diff
@@ -35,7 +35,9 @@ def __new__(cls, name: str, **kwargs):
 
                 instance = super().__new__(MultilingualCLIPModel)
             else:
-                raise ValueError(f'The CLIP model name=`{name}` is not supported.')
+                raise ValueError(
+                    f'Model {name} not found; available models = {list(_OPENCLIP_MODELS.keys()) + list(_MULTILINGUALCLIP_MODELS.keys())} '
```

Comment:
I would prefer to use `CLIP model name="{name}" ...` to highlight the name

Reply:
And what's more, we can improve the display by using rich table (or other api), e.g., 
```
- RN50
- RN16
- .....
```